opensourceleg/hardware/actuators.py

```diff
@@ -457,7 +457,7 @@ def __init__(
         self._mode: ActpackMode = self.control_modes.voltage
 
     def __repr__(self) -> str:
-        return f"DephyActpack[{self._name}]"
+        return f"{self._name}[DephyActpack]"
```

Comment:
Breaks unit tests. Unit tests need to be edited to reflect this change.

Reply:
My bad - should be good now.